lib/career_bot/responders/message_responder.rb

```diff
@@ -40,56 +52,66 @@ def run_postback
   end
 
   def start_apply_process
-    repository.update(conversation.id, question_index: 0,
+    repository.update(conversation_id,
+                      question_index: 0,
                       apply_job_shortcode: quick_reply.split('|').last)
-    ApplyResponder.new(conversation.id, message_text, {}).response
+    ApplyResponder.new(conversation_id, message_text, {}).response
+  end
+
+  def process_message
+    if message_data.entities.keys.any? { |key| key_exists(key) }
```

Comment:
Honestly speaking this line is quite unreadable for me <sadpepe.png>. Perhaps we could separate it to some other method? We could use `key_exists?` method - it's used only here.

Reply:
I was trying to make line shorter by that. My another idea is to move whole condition to other method (and it gonna work as well)